### DISCUSSION TITLE
No module named 'zipline.utils.calendars'

### DISCUSSION THREAD

**Discussion by OscarTheFE** [stefan-jansen/zipline-reloaded#170] [Q&A]
Hi all, 
Could any one help me out with the following issue.
I am able to do the following command, which makes me believe I have successfully downloaded zipline.reloaded package: 
"
from zipline.data import bundles
from zipline.pipeline import Pipeline
from zipline.pipeline.factors import AverageDollarVolume
"
But, when I try to import get_calendar function by "from zipline.utils.calendars import get_calendar," I got error message saying that "No module named 'zipline.utils.calendars'

I have zipline-reloaded version 2.4

**Comment by riverallzero** [comment]
Hi @OscarTheFE , i hope my answer will be helpful.
It's possible that the zipline package you installed is not complete or has been corrupted.
You can try uninstalling and reinstalling the zipline package to see if that resolves the issue. Do `pip install zipline` or `pip uninstall zipline` then `pip install zipline`
